### DISCUSSION TITLE
make username.github.io the reference page, not username.github.io/portfolio

### DISCUSSION THREAD

**Discussion by StefanBloemheuvel** [community/community#56383] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

I tried to create a portfolio page, which is now hosted at https://stefanbloemheuvel.github.io/portfolio/.
However, it would be nicer to have stefanbloemheuvel.github.io link to this portfolio repository github page.
Is that possible? It is a much cleaner link to share.

Kind regards,

Stefan

**Comment by airtower-luna** [comment, marked as answer]
For that you'll need to rename your repository to make it your "user site": https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites

The right name should be `stefanbloemheuvel.github.io` in your case.

**Reply by StefanBloemheuvel** [reply]
alright i just changed the repository called 'portfolio' into stefanbloemheuvel.github.io, it worked! thanks!
